src/e2e/app/cli/repo-tests/src/test_smart_transfer_protocol.rs

```diff
@@ -897,8 +897,8 @@ async fn test_smart_push_all_smart_pull_all(
             +--------+----+----------------------+----------------------+----------+-----------+-------+
             | offset | op | system_time          | match_time           | match_id | player_id | score |
             +--------+----+----------------------+----------------------+----------+-----------+-------+
-            | 2      | 0  | 2050-01-02T03:04:05Z | 2000-01-02T00:00:00Z | 2        | Charlie   | 90    |
-            | 3      | 0  | 2050-01-02T03:04:05Z | 2000-01-02T00:00:00Z | 2        | Alice     | 70    |
+            | 2      | 0  | 2050-01-02T03:04:05Z | 2000-01-02T00:00:00Z | 2        | Alice     | 70    |
+            | 3      | 0  | 2050-01-02T03:04:05Z | 2000-01-02T00:00:00Z | 2        | Charlie   | 90    |
```

Comment:
This is interesting. Looks like this test was relying on wrong ordering bug, as this root dataset's PK is `(match_id, player_id)`